x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/parser/ExpressionBuilder.java

```diff
@@ -167,8 +168,13 @@ public Expression visitOperatorExpressionDefault(EqlBaseParser.OperatorExpressio
         switch (predicate.kind.getType()) {
             case EqlBaseParser.SEQ:
                 return Predicates.combineOr(expressions(predicate.constant()).stream()
+                    .map(c -> new InsensitiveWildcardEquals(source, expr, c, zoneId))
```

Comment:
Is this correct? shouldn't this be for the `IN_INSESITIVE` and for `SEQ` the `InsensitiveEquals`?

Reply:
No. See the tests - `:` or SEQ supports wildcards while `in~` does not, hence why it is only insensitive.

Reply:
Oops, sorry, I confused myself.